tensorboard/compat/__init__.py

```diff
@@ -28,78 +28,92 @@
 import tensorboard.lazy as _lazy
 
 
-@_lazy.lazy_load('tensorboard.compat.tf')
+@_lazy.lazy_load("tensorboard.compat.tf")
 def tf():
-  """Provide the root module of a TF-like API for use within TensorBoard.
-
-  By default this is equivalent to `import tensorflow as tf`, but it can be used
-  in combination with //tensorboard/compat:tensorflow (to fall back to a stub TF
-  API implementation if the real one is not available) or with
-  //tensorboard/compat:no_tensorflow (to force unconditional use of the stub).
-
-  Returns:
-    The root module of a TF-like API, if available.
-
-  Raises:
-    ImportError: if a TF-like API is not available.
-  """
-  try:
-    from tensorboard.compat import notf  # pylint: disable=g-import-not-at-top
-  except ImportError:
+    """Provide the root module of a TF-like API for use within TensorBoard.
+
+    By default this is equivalent to `import tensorflow as tf`, but it can be used
+    in combination with //tensorboard/compat:tensorflow (to fall back to a stub TF
+    API implementation if the real one is not available) or with
+    //tensorboard/compat:no_tensorflow (to force unconditional use of the stub).
+
+    Returns:
+      The root module of a TF-like API, if available.
+
+    Raises:
+      ImportError: if a TF-like API is not available.
+    """
     try:
-      import tensorflow  # pylint: disable=g-import-not-at-top
-      return tensorflow
+        from tensorboard.compat import (
+            notf,
+        )  # pylint: disable=g-import-not-at-top
```

Comment:
Yeah. I suspect that the vast majority of our pylint comments are
useless, given that we basically only lint for syntax errors and free
variables. For instance, we have `line-too-long` on imports, but Google
style says that imports are unbounded-length, anyway, so that should be
superfluous.

I’ll go ahead and remove `line-too-long` and `import-not-at-top`s.

Reply:
Done in #3021 and #3022, which does resolve this.